docs/docs/guides/web3_eth/_category_.yml

```diff
@@ -0,0 +1,5 @@
+label: '📦 Web3-eth module'
```

Comment:
This should be consistent with package name `Web3Eth`
<img width="271" alt="image" src="https://github.com/web3/web3.js/assets/86780488/e564a30c-bb84-4505-98ac-ca61afc1606d">